go-apps/meep-auth-svc/server/auth-svc.go

```diff
@@ -432,25 +475,40 @@ func cacheFileserverPermissions(cfg *PermissionsConfig) {
 	authSvc.cache.Fileservers = make(map[string]*Permission)
 
 	for _, fs := range cfg.Fileservers {
-		// Cache fileserver permissions
+		// Create fileserver permissions
 		permission := new(Permission)
 		permission.Mode = fs.Mode
 		permission.Roles = make(map[string]string)
 		for role, access := range fs.Roles {
 			permission.Roles[role] = access
 		}
-		authSvc.cache.Fileservers[fs.Name] = permission
 
-		// Add auth service route
-		route := new(AuthRoute)
-		route.Name = fs.Name
-		route.Prefix = true
+		// Add auth service routes + cache filserver permissions
 		if fs.Sbox {
+			// Mep-specific sandbox fileservers
+			route := new(AuthRoute)
+			route.Prefix = true
+			route.Name = mepPrefix + fs.Name
+			route.Pattern = "/{sbox}/{mep}" + fs.Path
+			routes = append(routes, route)
+			authSvc.cache.Fileservers[route.Name] = permission
+
+			// Sandbox fileserver
+			route = new(AuthRoute)
+			route.Prefix = true
+			route.Name = fs.Name
 			route.Pattern = "/{sbox}" + fs.Path
+			routes = append(routes, route)
+			authSvc.cache.Fileservers[route.Name] = permission
 		} else {
+			// Global fileserver
```

Comment:
some common code could be taken out of if/else

Reply:
Agreed. Will try to address when working in this module again.